tests/integration/tier0/non-destructive/rhsm-facts/main.fmf

```diff
@@ -0,0 +1,20 @@
+summary+: |
+    RHSM facts update when in analysis mode
+description+: |
+    Verify that the subscription-manager facts command is called after the
+    analysis is done (either succeeding or failling).
+
+/rhsm_facts_update_in_analysis:
+    summary+: |
+        RHSM facts update after analysis
+    description+: |
+        This test verifies that the RHSM facts are uploaded when running
+        convert2rhel with analysis option.
```

Comment:
This is just duplicating the summary and description from the root of the path in the file.
Could be consolidated, I wouldn't block the merge on this though.